tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivativeImpl.tpp

```diff
@@ -1348,6 +1348,54 @@ void test_impl(const Spectral::Quadrature quadrature,
 template <SystemType system_type, UseBoundaryCorrection use_boundary_correction,
           size_t Dim>
 void test() noexcept {
+  // The test impl is structured in the following way:
+  // - the static mesh volume contributions are computed "by-hand" and used more
+  //   or less as a regression test. This is relatively easy for Gauss-Lobatto
+  //   points, a bit more tedious for Gauss points. We also assume the solution
+  //   is constant in the y & z direction to make the math easier. The math
+  //   implemented and checked against are the static mesh contributions coded
+  //   up in TimeDerivativeTerms, with the addition of the strong or weak flux
+  //   divergence that are computed by generic code outside the
+  //   TimeDerivativeTerms struct.
+  //
+  //   The conservative equations are:
+  //     dt var1 = -d_i (var1**2 + var2^i) + var3**2
+  //     dt var2^j = -d_i (var1 * var2^i * var2^j + delta^i_j var1**3) +var3 * j
+  //   if there are primitive variables then:
+  //     dt var1 += prim_var1
+  //
+  //   The non-conservative equations are:
+  //     dt var1 = -var2^i d_i var1 + var3**2
+  //     dt var2^j = -var1 * var2^i d_i var2^j + var3 * j
+  //
+  //   The mixed conservative non-conservative equations are:
+  //     dt var1 = -var2^i d_i var1 + var3**2
+  //     dt var2^j = -d_i (var1 * var2^i * var2^j + delta^i_j var1**3) +var3 * j
+  //   if there are primitive variables then:
+  //     dt var1 += prim_var1
+  //
+  //   The mesh only has 2 points per dimension to make the math not too
+  //   terrible. The differentiation matrices & weights for Gauss points are:
+  //    D^{strong}: {{-0.866025403784438, 0.866025403784438},
+  //                 {-0.866025403784438, 0.866025403784438}}
+  //    w: {1.0, 1.0}
+  //    D^{weak}: {{-0.866025403784438, -0.866025403784438},
+  //               {0.866025403784438, 0.866025403784438}}
+  //
+  //   Gauss-Lobatto:
+  //    D^{strong}: {{-1/2, 1/2}, {-1/2, 1/2}}
+  //    w: {1.0, 1.0}
+  //    D^{weak}: {{-1/2, -1/2}, {1/2, 1/2}}
+  //
+  //   The weak divergence matrix is given by:
+  //     D_{i,l}^{weak} = (w_l/w_i) D_{l,i}^{strong}
+  //
+  // - the additional parts, such as moving mesh and boundary corrections become
+  //   increasingly tedious (and unmaintainable) to work out separately
+  //   (implementing the entire moving-mesh DG algorithm in Python is _a lot_ of
+  //   work and extra code), so instead we verify that the expected mathematical
+  //   manipulations occur.
```

Comment:
Great comment, thank you!

I'd just suggest expanding the second bullet by adding this sentence at the end:
"Specifically, we check adding the mesh velocity to the fluxes, and lifting the boundary contributions to the volume, even though the mesh velocity and boundary contributions are not the correct DG values"